docs/src/user_guide/step3_input_data.md

```diff
@@ -0,0 +1,56 @@
+# Step 3: Preparing the input data
+
+As mentioned before, the input data can be classified into two types:
+
+ - Meteorological forcing: maps with timeseries for each model pixel, with values for
+   precipitation, temperature, and potential evaporation. This data should be provided as a
+   three-dimensional dataset, with the `x`, `y` and `time` dimensions. 
+ - Static maps.
+
+
+## Meterological data
+
+Meteorological data is provided as a single NetCDF file, with several variables containing
+the forcing data for precipitation, temperature and potential evaporation. The code snippet
+below shows the contents of the example file (downloaded [here](@ref wflow_sbm_data)), and
+opened with `xarray` in Python. As can be seen, each variable consists of a
+three-dimensional dataset (`lon`, `lat`, and `time`), and each timestep consists of a
+two-dimensional map with values at each gridcell. Only values within the basin are required.
+
+```
+<xarray.Dataset>
+Dimensions:  (lon: 291, lat: 313, time: 366)
+Coordinates:
+  * lon      (lon) float64 5.429 5.438 5.446 5.454 ... 7.821 7.829 7.837 7.846
+  * lat      (lat) float32 50.42 50.41 50.4 50.4 ... 47.85 47.84 47.83 47.82
+  * time     (time) datetime64[ns] 2000-01-01 2000-01-02 ... 2000-12-31
+Data variables:
+    P        (time, lat, lon) float32 ...
+    TEMP     (time, lat, lon) float32 ...
+    PET      (time, lat, lon) float32 ...
+```
+
+## Static data
+
+
+### List of essential static data
```

Comment:
I did change this list because it also contained some non-essential static data that is not required for a Wflow run, but can provide additional useful info (for example streamorder). Also the required maps (not optional) depend on the model type. Perhaps we should mention here only static data that is required for one wflow model configuration, for example wflow_sbm + kinematic wave? In that case the wflow_dem is also not required.

Reply:
Fixed, and added a link to the example models